src/vrdev/src/Scatterplot.ts

```diff
@@ -0,0 +1,57 @@
+//run tsc Scatterplot.ts to compile into Scatterplot.js file
+
+// for running on browser
+// import * as _d3 from "d3";
+
+// declare global {
+//   const d3: typeof _d3;
+// }
+
+//for unit testing
+import * as d3 from "d3";
+
+
+export class Scatterplot{
+    data: number[];
+    shape: string;
+
+  constructor(){
+    this.data = [0, 0];
+  }
+  init(data: number[]){
+    // create a scale so that there is correspondence between data set and screen render
+    this.data = data;
+    
+    this.generatePts("a-sphere");  
+    this.setColor("blue");
+    //console.log("hello");
+    return this.data;
```

Comment:
Probably no need to return `data`.

Reply:
I return `data` for testing purposes. If I make `data` `private`, I'll have to return `data` bc that is the only way I can access the variable's value. I could always make a function specifically made to retrieve `data`, but dont know if that would be convoluting too much